lib/settings.py

```diff
@@ -124,20 +121,18 @@ def write_to_file(data_to_write, filename, mode="a+"):
     return filename
 
 
-def load_api_keys(path="{}/etc/tokens".format(os.getcwd())):
+def load_api_keys(path="{}/etc/tokens".format(CUR_DIR)):
 
     """
     load the API keys from their .key files
     """
 
-    def makedir(dir):
-        """
-        make the directory if it does not exist
-        """
-        if not os.path.exists(dir):
-            os.mkdir(dir)
```

Comment:
I think, two small lines will be better than one-time call of locally defined function (in case of optimization too).

Reply:
Okay I see what you did, change the variable to path so it doesn't overwrite the built in `dir` var